python_modules/dagster-test/dagster_test/toys/partitioned_assets/dynamic_asset_partitions.py

```diff
@@ -28,6 +30,17 @@ def dynamic_partitions_asset2(dynamic_partitions_asset1):
     partitions_def=customers_partitions_def,
 )
 
+multipartition_w_dynamic_partitions_def = MultiPartitionsDefinition(
+    {"customers": customers_partitions_def, "daily": DailyPartitionsDefinition("2023-01-01")}
+)
+
+
+@asset(
+    partitions_def=multipartition_w_dynamic_partitions_def,
+)
+def dynamic_dimension_success():
```

Comment:
It would be helpful to give this a name that describes what it is - e.g. `multi_partitioned_with_dynamic_dimension`